requirements.txt

```diff
@@ -24,8 +24,7 @@
 
 # Top-level requirements for shasta-install-utility-common
 
-# This should be pinned exactly to 1.3.2 due to issues with the build of cray-product-catalog
-cray-product-catalog==1.3.2
+cray-product-catalog==1.8.5
```

Comment:
cray-product-catalog PR is out and needs to merge first, so bump this to latest version after that releases.